src/Reader/readpar.f90

```diff
@@ -2772,7 +2775,9 @@ SUBROUTINE readpar_output(EQN,DISC,IO,MPI,CalledFromStructCode)
           ENDIF
       END IF
 
-      ALLOCATE(IO%IntegrationMask(9),STAT=allocstat )                        !
+    IO%OutputGroups = pack(OutputGroups, OutputGroups >= 0)
+
+	  ALLOCATE(IO%IntegrationMask(9),STAT=allocstat )                        !
```

Comment:
fix code alignment

Reply:
Also don't you need to allocate IO%OutputGroups first?

Reply:
I don't think so - pack seems to allocate. All examples that I can find where pack is used on allocateable arrays do not use allocate manually.

Reply:
ok you're right (I was too lazy to google).